.gitmodules

```diff
@@ -5,3 +5,7 @@
 [submodule "3rdparty/treelite"]
 	path = 3rdparty/treelite
 	url = https://github.com/neo-ai/treelite
+[submodule "3rdparty/json"]
+	path = 3rdparty/json
+	url = https://github.com/nlohmann/json.git
```

Comment:
Just wondering if there are any licensing issues using this new `nlohmann/json.git` git directory?

Reply:
According to some google searches the MIT license used by nlohmann is compatible with the Apache License 2.0. But we should check internally to be safe.

Reply:
MIT License is pre-approved.

https://inside.amazon.com/en/services/legal/us/OpenSource/Pages/BlessedOpenSourceLicenses.aspx

Reply:
Could we just add single `nlohmann/json.hpp` file to our include dir?

Reply:
I found that TVM already has a library "picojson" we could use instead https://github.com/neo-ai/tvm/tree/dev/3rdparty/picojson

Reply:
Nice! BTW, This PR should be rebased to use the latests `3rdparty/tvm`

Reply:
By looking at picojson it seems more complicate to use compared to nlohmann. What do you think?  Also the https://github.com/nlohmann/json/commits/develop is begin contributed lot more actively.

Reply:
> Could we just add single `nlohmann/json.hpp` file to our include dir?

We are adding a single_include directory. I am not sure what you mean?